### PR TITLE
Wordsets

### PR DESCRIPTION
**Pull request by ahurbol**

### CODE REVIEW

File: navgation/WordSetStack.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+};
+
+const Stack = createNativeStackNavigator<WordSetStackParamList>();
+
+const WordSetStack = () => {
+  return (
+    <WordSetProvider navigation={useNavigation()}>
+      <Stack.Navigator screenOptions={{ headerShown: false }}>
+        <Stack.Screen name={"List"} component={WordSetsListScreen} />
+        <Stack.Screen name={"Display"} component={WordSetDisplayScreen} />
```
Comment (paliwodam) [inline navgation/WordSetStack.tsx:21]: I would recommend to name it  "DisplayWordSet"

File: navgation/WordSetStack.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  FlashCards: undefined;
+};
+
+const Stack = createNativeStackNavigator<WordSetStackParamList>();
+
+const WordSetStack = () => {
+  return (
+    <WordSetProvider navigation={useNavigation()}>
+      <Stack.Navigator screenOptions={{ headerShown: false }}>
+        <Stack.Screen name={"List"} component={WordSetsListScreen} />
```
Comment (paliwodam) [inline navgation/WordSetStack.tsx:20]: I would recommend naming it  "ListWordSet"

**Review by paliwodam — APPROVED** [review]
A very fine piece of code:)

**Review by MichalGniadek — APPROVED** [review]
LGTM :)

**Pull request merged**